docs/write_driver.md

```diff
@@ -324,6 +324,21 @@ To test a GraphQL API, in the the `getProblemInfo()`, you need to return an inst
 Here, you need to specify the endpoint of where the GraphQL API can be accessed. Default value is `/graphql`.
 Note: must be able to do an _introspective query_ on such API to fetch its schema. If this is disabled for security reasons, _EvoMaster_ will fail.  
 
+## RPC APIs
+
+To test RPC APIs, in the the `getProblemInfo()`, you need to return an instance of the
+`RPCProblem` class.
+Fuzzing RPC APIs requires a driver, and importing a JVM version of the API client library used to communicate with it.
```

Comment:
append

_(Note that such API client library could be generated based on schema definition file, e.g., `.proto` and `.thrift`)_

?